File: java/FizzBuzz.java
package com.nksm.tsukumo;

public class FizzBuzz {
	
	static private String calculate(Integer number) {
	        if (number % 3 == 0 && number % 5 == 0) {
	        	return  "FizzBuzz";
	        }
	        if (number % 3 == 0) {
	        	return  "Fizz";
	        }
	        if (number % 5 == 0) {
	            return  "Buzz";
	        }
			return number.toString();
	}
	
	public static void main(String[] args) {
		long start = System.currentTimeMillis();

		for(Integer i = 0 ;i <= 1000000;i++ ) {
			System.out.println(calculate(i));
			
		}
		long end = System.currentTimeMillis();
		System.out.println((end - start)  + "ms");
	}
}

File: ruby/fizzbuzz.rb
require 'time'

start = Process.clock_gettime(Process::CLOCK_MONOTONIC)

range = 1..1000000
range.each{|num|
    if ((num % 3 == 0) && (num % 5 == 0)) then
        puts 'fizzbuzz'
    elsif (num % 3 == 0) then
        puts 'fizz'
    elsif (num % 5 == 0) then
        puts 'buzz'
    else
        puts '%d' % [num]
    end
}

finish = Process.clock_gettime(Process::CLOCK_MONOTONIC)

elapsed = finish - start
puts elapsed